app/models/proceeding.rb

```diff
@@ -42,13 +44,37 @@ def domestic_abuse?
     ccms_matter_code == "MINJN"
   end
 
-  # def default_level_of_service_level
-  #   "3"
-  # end
-  #
-  # def default_level_of_service_name
-  #   "Full Representation"
-  # end
+  def substantive_scope_limitations
+    scope_limitations.where(scope_type: :substantive)
+  end
+
+  def emergency_scope_limitations
+    scope_limitations.where(scope_type: :emergency)
+  end
```

Comment:
[similarly](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/4324/files#r977574055)